client/state/action-types.js

```diff
@@ -44,6 +44,8 @@ export const ANALYTICS_EVENT_RECORD = 'ANALYTICS_EVENT_RECORD';
 export const ANALYTICS_MULTI_TRACK = 'ANALYTICS_MULTI_TRACK';
 export const ANALYTICS_PAGE_VIEW_RECORD = 'ANALYTICS_PAGE_VIEW_RECORD';
 export const ANALYTICS_STAT_BUMP = 'ANALYTICS_STAT_BUMP';
+export const ANALYTICS_CONTINOUS_MONITOR_ON = 'ANALYTICS_CONTINOUS_MONITOR_ON';
+export const ANALYTICS_CONTINOUS_MONITOR_OFF = 'ANALYTICS_CONTINOUS_MONITOR_OFF';
```

Comment:
The action types are named quite differently from the reducer and selector -- ideally they should be a bit more consistent... (Doesn't need to be something with `CONTINUOUS_TRACKING`, also cool to change the reducer/selector name instead.)

(Prefixing them with `ANALYTICS_` is a good thing 👍 )